app/javascript/packages/document-capture/components/_file-input.scss

```
@@ -17,6 +17,18 @@
   border-width: 3px;
 }
 
+usa-file-input:not(
```

I'm assuming this is meant to be a CSS class selector, but it's currently implemented as an element selector, so it's probably not working as intended.

```suggestion
.usa-file-input:not(
```